docs/index-en.html

```diff
@@ -0,0 +1,4245 @@
+          </li>
+          <li> <a href="#phase" title="https://w3id.org/ontouml#phase">phase</a>
+          </li>
+          <li> <a href="#phaseMixin" title="https://w3id.org/ontouml#phaseMixin">phaseMixin</a>
+          </li>
+          <li> <a href="#quality" title="https://w3id.org/ontouml#quality">quality</a>
+          </li>
+          <li> <a href="#qualityNature" title="https://w3id.org/ontouml#qualityNature">qualityNature</a>
+          </li>
+          <li> <a href="#quantity" title="https://w3id.org/ontouml#quantity">quantity</a>
+          </li>
+          <li> <a href="#quantityNature" title="https://w3id.org/ontouml#quantityNature">quantityNature</a>
+          </li>
+          <li> <a href="#relator" title="https://w3id.org/ontouml#relator">relator</a>
+          </li>
+          <li> <a href="#relatorNature" title="https://w3id.org/ontouml#relatorNature">relatorNature</a>
+          </li>
+          <li> <a href="#role" title="https://w3id.org/ontouml#role">role</a> </li>
+          <li> <a href="#roleMixin" title="https://w3id.org/ontouml#roleMixin">roleMixin</a>
+          </li>
+          <li> <a href="#shared" title="https://w3id.org/ontouml#shared">shared</a>
+          </li>
+          <li> <a href="#situation" title="https://w3id.org/ontouml#situation">situation</a>
+          </li>
+          <li> <a href="#situationNature" title="https://w3id.org/ontouml#situationNature">situationNature</a>
+          </li>
+          <li> <a href="#subCollectionOf" title="https://w3id.org/ontouml#subCollectionOf">subCollectionOf</a>
+          </li>
+          <li> <a href="#subkind" title="https://w3id.org/ontouml#subkind">subkind</a>
+          </li>
+          <li> <a href="#subQuantityOf" title="https://w3id.org/ontouml#subQuantityOf">subQuantityOf</a>
+          </li>
+          <li> <a href="#termination" title="https://w3id.org/ontouml#termination">termination</a>
+          </li>
+          <li> <a href="#triggers" title="https://w3id.org/ontouml#triggers">triggers</a>
+          </li>
+          <li> <a href="#type" title="https://w3id.org/ontouml#type">type</a> </li>
+          <li> <a href="#typeNature" title="https://w3id.org/ontouml#typeNature">typeNature</a>
+          </li>
+        </ul>
+      </div>
+      <!--DESCRIPTION SECTION-->
+      <div id="description">
+        <h2 id="desc" class="list">OntoUML Metamodel Vocabulary: Description <span
+
+            class="backlink"> back to <a href="#toc">ToC</a></span></h2>
+        <span class="markdown"> This is a placeholder text for the description
+          of your ontology. The description should include an explanation and a
+          diagram explaining how the classes are related, examples of usage,
+          etc.</span> </div>
+      <!--CROSSREF SECTION-->
+      <div id="crossref">
+        <h2 id="crossreference" class="list">Cross-reference for OntoUML
+          Metamodel Vocabulary classes, object properties and data properties <span
```

Comment:
Could we change this title? It is a bit cryptic.

Reply:
Done.

Reply:
The problem was with the "cross-reference" part. Why not call it simply "Vocabulary specification" as in [DCAT](https://www.w3.org/TR/vocab-dcat-2/#vocabulary-specification)?

Reply:
Done.